sdk/keyvault/azure.keyvault.secrets.txt

```diff
@@ -0,0 +1,57 @@
+**** Package/Module azure.keyvault.secrets****
+class SecretClient(KeyVaultClientBase, object):
+	vault_url:str
+	credential
+
+	vault_url
+
+	def __init__(self, vault_url: str, credential, transport=None, api_version=None, **kwargs): ...
+	def backup_secret(self, name: str, **kwargs) -> bytes: ...
+	def delete_secret(self, name: str, **kwargs) -> ~azure.keyvault.secrets.models.DeletedSecret: ...
+	def get_deleted_secret(self, name: str, **kwargs) -> ~azure.keyvault.secrets.models.DeletedSecret: ...
+	def get_secret(self, name: str, version: str=None, **kwargs) -> ~azure.keyvault.secrets.models.Secret: ...
+	def list_deleted_secrets(self, **kwargs) -> ~azure.core.paging.ItemPaged[~azure.keyvault.secrets.models.DeletedSecret]: ...
+	def list_secret_versions(self, name: str, **kwargs) -> ~azure.core.paging.ItemPaged[~azure.keyvault.secrets.models.SecretProperties]: ...
+	def list_secrets(self, **kwargs) -> ~azure.core.paging.ItemPaged[~azure.keyvault.secrets.models.SecretProperties]: ...
+	def purge_deleted_secret(self, name: str, **kwargs): ...
+	def recover_deleted_secret(self, name: str, **kwargs) -> ~azure.keyvault.secrets.models.SecretProperties: ...
+	def restore_secret(self, backup: bytes, **kwargs) -> ~azure.keyvault.secrets.models.SecretProperties: ...
+	def set_secret(self, name: str, value: str, content_type: str=None, enabled: bool=None, not_before: datetime.datetime=None, expires: datetime.datetime=None, tags: dict=None, **kwargs) -> ~azure.keyvault.secrets.models.Secret: ...
```

Comment:
Is there any way to minimize the number of positional parameters here? E.g. move some of the optional flags to `**kwargs`?

Reply:
Yes we could move everything but name and value to kwargs if this is what we want to do for python across the board.